modules/dhcp_common/isc/omapi_provider.rb

```diff
@@ -1,10 +1,9 @@
 require 'time'
-require 'dhcp_isc/isc_common'
+require 'dhcp_common/server'
 
-module Proxy::DHCP::ISC
```

Comment:
```::Proxy::DHCP::ISC``` namespace is reserved for the dhcp isc provider and can't be used for shared bits of code.